app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -101,10 +101,13 @@ class PatientSummaryUpdate : Update<PatientSummaryModel, PatientSummaryEvent, Pa
 
   private fun contactDoctorClicked(model: PatientSummaryModel): Next<PatientSummaryModel, PatientSummaryEffect> {
     return when (model.teleconsultInfo) {
+      // TODO (SM): Check phone numbers before dispatching
+      //  LoadPatientTeleconsultationInfo to show contact doctor sheet
       is TeleconsultInfo.Fetched -> dispatch(LoadPatientTeleconsultationInfo(
           model.patientUuid,
           model.patientSummaryProfile?.bpPassport,
-          model.currentFacility
+          model.currentFacility,
+          model.teleconsultInfo.phoneNumbers.first()
```

Comment:
Since the API currently does not return any numbers, will this cause a problem if we merge it in?

Reply:
Yes, I will check with server devs on when the API will be available. Another option is to support both the old phone number and the new phone numbers list. Which might make it a bit complex.

Reply:
Confirmed with server devs, at latest the API should be available on `QA` & `SBX` on Friday.

Reply:
We would temporarily make it work by coercing the primary phone number to a list with a single element if the list of phone numbers is empty or not present.

Reply:
Yeah, I have done that change in the effect handler where we are fetching the phone numbers.